scripts/income_limits/util.py

```diff
@@ -0,0 +1,350 @@
+import datetime
+import json
+import numpy as np
+import pandas as pd
+import pathlib
+import re
+import requests
+import time
+
+# -- Globals -- #
+STATE_POSTAL_TO_GEOID = {
+    "AL": "01",
+    "AK": "02",
+    "AZ": "04",
+    "AR": "05",
+    "CA": "06",
+    "CO": "08",
+    "CT": "09",
+    "DE": "10",
+    "FL": "12",
+    "GA": "13",
+    "HI": "15",
+    "ID": "16",
+    "IL": "17",
+    "IN": "18",
+    "IA": "19",
+    "KS": "20",
+    "KY": "21",
+    "LA": "22",
+    "ME": "23",
+    "MD": "24",
+    "MA": "25",
+    "MI": "26",
+    "MN": "27",
+    "MS": "28",
+    "MO": "29",
+    "MT": "30",
+    "NE": "31",
+    "NV": "32",
+    "NH": "33",
+    "NJ": "34",
+    "NM": "35",
+    "NY": "36",
+    "NC": "37",
+    "ND": "38",
+    "OH": "39",
+    "OK": "40",
+    "OR": "41",
+    "PA": "42",
+    "RI": "44",
+    "SC": "45",
+    "SD": "46",
+    "TN": "47",
+    "TX": "48",
+    "UT": "49",
+    "VT": "50",
+    "VA": "51",
+    "WA": "53",
+    "WV": "54",
+    "WI": "55",
+    "WY": "56",
+    "DC": "11",
+    "AS": "60",
+    "GU": "66",
+    "MP": "69",
+    "PR": "72",
+    "VI": "78"
+}
+# Note that while PR is a territory, census data for PR is stored
+# with the other states and DC so it does not require special handling
+TERRITORY_POSTAL_CODES = ['AS', 'GU', 'MP', 'VI']
+
+# HUD API
+with (pathlib.Path() / 'scripts' / 'income_limits' / 'config.json').open('r') as f:
+    config = json.load(f)
+HUD_API_HEADERS = {"Authorization": f"Bearer {config['hud_api_key']}"}
```

Comment:
Could you document this in the readme? Mention the file path, show the structure: `{"hud_api_key": "..."}`, and say RA employees should get the key from 1Password.